ccd-definition/CaseEventToFields.json

```diff
@@ -396,5 +396,16 @@
     "PageID": 1,
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "CaseEventID": "uploadStandardDirections",
+    "CaseFieldID": "standardDirections",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL",
```

Comment:
I would consider this mandatory. No point submitting event with one field without providing value. If user selected even by accident they should cancel it.